domains/pallets/domain-tracker/src/mock.rs

```diff
@@ -15,7 +15,7 @@ frame_support::construct_runtime!(
         UncheckedExtrinsic = UncheckedExtrinsic,
     {
         System: frame_system::{Pallet, Call, Config, Storage, Event<T>},
-        DomainTracker: crate::{Pallet, Call, Storage, Event<T>},
+        DomainTracker: crate::{Pallet, Storage, Event<T>},
```

Comment:
```suggestion
        DomainTracker: crate,
```

You can directly use the pallet and the components necessary will be expanded automatically so that you don't have to update when there is a change to it.